mathics/builtin/atomic/symbols.py

```diff
@@ -238,100 +303,10 @@ class Definition(Builtin):
     attributes = A_HOLD_ALL | A_PROTECTED
     summary_text = "give values of a symbol in a form that can be stored in a package"
 
-    def format_definition(self, symbol, evaluation, grid=True):
+    def format_definition(self, symbol, evaluation: Evaluation, grid: bool = True):
```

Comment:
Please add `-> SymbolNull` since this is what this returns.

Reply:
Done.